docs/administration-guide/06-collections.md

```diff
@@ -0,0 +1,44 @@
+## Creating Collections for Your Saved Questions
+---
+
+Collections are a great way to organize your saved questions and decide who gets to see and edit things. Collections could be things like, "Important Metrics," "Marketing KPIs," or "Questions about users." Multiple user groups can be given access to the same collections, so we don't necessarily recommend naming collections after user groups.
```

Comment:
Very minor, but might be worth linking to the documentation on user groups here just in case someone doesn't know what a user group is or wants to learn more.